src/serverHistory.ts

```diff
@@ -45,8 +45,12 @@ class ServerHistory implements History {
   complete() {
     this._completeCallback();
   }
+
+  getCurrentLocation() {
+    return this.currentLocation;
+  }
 }
 
-export function createServerHistory(): History {
-  return new ServerHistory();
+export function createServerHistory(currentLocation: Location | Pathname): History {
+  return new ServerHistory(createLocation(currentLocation));
```

Comment:
Good point I forgot about that! However it's always going to be at the pathname `/`which is probably not what the developer expects